tests/cli/test_ftable.py

```diff
@@ -58,6 +58,10 @@ def test_multiline(self) -> None:
 
 
 class TestTable:
+    def test_empty(self) -> None:
+        result = list(table(()))
+        assert len(result) == 0
```

Comment:
```suggestion
        assert result == []
```

Reply:
Wait, table returns an Iterator[str], is it correct to check that it's an empty list?

Reply:
But `result` is a list.

Reply:
Oops, sorry, missed that